pkg/scheduler/internal/cache/cache.go

```diff
@@ -743,12 +743,11 @@ func (cache *schedulerCache) cleanupAssumedPods(now time.Time) {
 			klog.Fatal("Key found in assumed set but not in podStates. Potentially a logical error.")
 		}
 		if !ps.bindingFinished {
-			klog.V(5).Infof("Couldn't expire cache for pod %v/%v. Binding is still in progress.",
-				ps.pod.Namespace, ps.pod.Name)
+			klog.V(5).InfoS("Couldn't expire cache for pod. Binding is still in progress.", "pod", klog.KObj(ps.pod))
 			continue
 		}
 		if now.After(*ps.deadline) {
-			klog.Warningf("Pod %s/%s expired", ps.pod.Namespace, ps.pod.Name)
+			klog.InfoS("Pod expired", "pod", klog.KObj(ps.pod))
```

Comment:
Do not change log level.

Reply:
Based on the comments above (https://github.com/kubernetes/kubernetes/pull/97612/files#r550191021) this is the same issue, that a structured warning function is not currently available.

The linked comments seem to imply that low-level InfoS is semantically equivalent to a warning, at least until that function is available. Same applies to the other comments in this PR